t/lib/t/MusicBrainz/Server/Controller/URL/Show.pm

```diff
@@ -15,7 +15,7 @@ my $c    = $test->c;
 MusicBrainz::Server::Test->prepare_test_database($c, '+url');
 
 $mech->get_ok('/url/25d6b63a-12dc-41c9-858a-2f42ae610a7d');
-$mech->content_contains('http://zh-yue.wikipedia.org/wiki/%E7%8E%8B%E8%8F%B2');
+$mech->content_contains('http://zh-yue.wikipedia.org/wiki/王菲');
 $mech->content_contains('zh-yue: 王菲');
```

Comment:
Reversal of the test, as I warned before. :) I left the additional test, though.

BTW, the full URL in decoded form that the test checks for is in the heading [here](http://chirlu.mbsandbox.org/url/25d6b63a-12dc-41c9-858a-2f42ae610a7d), it was never in an `href`.